deps.qt/qt6.ps1

```diff
@@ -34,6 +34,10 @@ function Setup {
         '--force'
     )
     Invoke-External perl init-repository @Options
+
+    if ( (${Target} -eq "arm64") -and ($Env:Qt6_DIR -eq $null) ) {
```

Comment:
Do not rely on an environment variable, add an explicit build script parameter that needs to be supplied upon invocation (e.g. `QtHostInstall`) which is checked for and used instead.